src/main/java/seedu/address/model/transaction/Debt.java

```diff
@@ -0,0 +1,94 @@
+package seedu.address.model.transaction;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+/**
+ * Represents a Debt.
+ * Guarantees: immutable.
```

Comment:
[Question] Is the debt object considered immutable if you can add debts and remove debts?

Reply:
It's only immutable if the object itself is not able to be modified, so if you're using it it doesn't count

Reply:
Thanks for answering my question!